### PR TITLE
added 2 new use cases from Ahsan Kareem

### PR DESCRIPTION
**Pull request by silviamazzoni**

### CODE REVIEW

File: user-guide/docs/usecases/workinprogress.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+{% include-markdown 'kareem/usecase2.md' %}
+
+<hr style="border: dashed white 1.0px;background-color: white;height: 200.0px;"/>
+---
+
+<!-- ##  Tamkang Database -->
+
+{% include-markdown 'kareem/usecase3.md' %}
+
+<hr style="border: dashed white 1.0px;background-color: white;height: 200.0px;"/>
```
Comment (wesleyboar) [inline user-guide/docs/usecases/workinprogress.md]: This solution has drawbacks —

- redundant `<hr>`s
- duplicate complex code
- inline styles

— so please [use a stylesheet instead](https://github.com/DesignSafe-CI/DS-User-Guide/pull/18).

<sup>To learn more, see "Notes" on [#18](https://github.com/DesignSafe-CI/DS-User-Guide/pull/18).</sup>

File: user-guide/mkdocs.yml [hunk 2]
```diff
@@ -89,6 +89,7 @@ nav:
     - GeoHazard: usecases/geohazardusecases.md
     - Seismic: usecases/seismicusecases.md
     - Wind and Storm Surge: usecases/windstormsurgeusecases.md
+    - Work in Progress: usecases/workinprogress.md
```
Comment (wesleyboar) [inline user-guide/mkdocs.yml]: Generally, a "Work in Progress" in not something deployed. A merge to `main` deploys this. If your intent is to test, is your test complete? (It worked for me.)

File: user-guide/docs/usecases/windstormsurgeusecases.md [hunk 3]
```diff
@@ -1,41 +1,59 @@
...
+<!--- end of comment -->
+
+<!--- <!-- ##  Tamkang Database -->
+
+<!--- Silvia Mazzoni 5/29/2024. remove these comments after completion -->
+<!--- {% include-markdown 'kareem/usecase3.md' %} -->
+<!---  -->
+<!--- <hr style="border: dashed white 1.0px;background-color: white;height: 200.0px;"/> -->
+<!--- --- -->
+<!--- end of comment -->
```
Comment (wesleyboar) [inline user-guide/docs/usecases/windstormsurgeusecases.md]: <sub>Just a single set of `<!--` (at start) and `-->` (at end) should be adequate. I'll assume this verbose commenting method is a side-effect of code editor keyboard shortcut or something.</sub>

File: user-guide/docs/usecases/DesignsafeLogo.jpg
Comment (wesleyboar) [inline user-guide/docs/usecases/DesignsafeLogo.jpg:1]: If this ends up not being used, please remember to remove.

**Review by wesleyboar — CHANGES_REQUESTED** [review]
Remove any unused assets. Evaluate whether you intend to deploy "Work in Progress". Evaluate a different solution for greater space above `<hr>`'s: #18.

File: user-guide/docs/usecases/windstormsurgeusecases.md (hunk 3 above)
Comment (silviamazzoni) [reply user-guide/docs/usecases/windstormsurgeusecases.md]: yes of course, but I was marking it as a block. easier to work with it that way.

File: user-guide/mkdocs.yml (hunk 2 above)
Comment (wesleyboar) [reply user-guide/mkdocs.yml]: @silviamazzoni and I chatted. WIP can be removed, yet pages are still accessible at URL:

| delete | `/user-guide/usecases/kareem/usecase` | `/user-guide/usecases/kareem/usecase2` | `/user-guide/usecases/kareem/usecase3` |
| - | - | - | - |
| deleted wip test | usecase | usecase2 | usecase3 |

**Review by wesleyboar — APPROVED** [review]
Great, thanks. I will merge, then resolve the problem[^1] my direction from #18 causes.

[^1]: Any `<hr>`'s directly after an `<h1>` are also receiving the super-spaced `hr` styles.

**Pull request merged**